Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+ruby '2.4.1' if ENV['DYNO']
```

Comment:
Yeah don't know on that. I feel like i've seen it both ways. I'll kill the if since it seems we don't do it on projects with more attention (eg: api)